lib/galaxy/tools/search/__init__.py

```diff
@@ -130,15 +133,17 @@ def search(self, q, tool_name_boost, tool_section_boost, tool_description_boost,
             )
         )
         # Set query to search name, description, section, help, and labels.
-        parser = MultifieldParser(['name', 'description', 'section', 'help', 'labels', 'stub'], schema=self.schema)
+        og = OrGroup.factory(0.9)
```

Comment:
@martenson can we add a comment here and explain what the factory and the 0.9 are doing.
"The Or query will score documents higher that contain multiple of the query term/token. The 0.9 is a scaling factor that indicates how much better a document is rated. "

Reply:
docs added